street_level_crime/create_csv.py

```diff
@@ -7,13 +9,52 @@ def convert_to_csv(headers: tuple, data: list) -> csv:
     :Tuple headers: Contians the headers of the csv file
     :List data: Contains a list of row data in format List[List[String]]
     """
+    print("When entering a filename please don't add .csv to the file")
+    filename = input("Please enter a filename: ")
```

Comment:
Use the ui_helpers.py methods to get input

Reply:
Explain what the file path is used for

Reply:
This method should take a path as a parameter, instead of requesting input

Reply:
Done